src/components/modals/KeyResultModal.vue

```diff
@@ -94,9 +98,3 @@ export default {
   },
 };
 </script>
-
-<style lang="scss" scoped>
-::v-deep .modal {
-  max-width: 350px;
```

Comment:
Tenker vi at alle modaler nødvendigvis bør/må ha samme bredde (om ikke bare modaler av tilsvarende type, som f.eks. denne og `ProgressModal`)?

Reply:
Ja, tenkte i utgangspunktet at de burde det. Eller at de i hvert fall ikke trenger å overstyre det lokalt, men at vi heller kan lage flere klasser av modaler da, hvis vi ser at vi trenger det. Type `wide`, `narrow`, etc?

Reply:
Ja, absolutt. Hadde bare en utfordring med inlining av datepicker, for å unngå "modal i modal", som krever en spesifikk bredde (ikke veldig responsiv), men det løser seg.